core/src/main/java/org/bitcoinj/base/Bech32.java

```diff
@@ -128,8 +126,10 @@ public static String encode(final Bech32Data bech32) {
 
     /** Encode a Bech32 string. */
     public static String encode(Encoding encoding, String hrp, final byte[] values) {
-        checkArgument(hrp.length() >= 1, "Human-readable part is too short");
-        checkArgument(hrp.length() <= 83, "Human-readable part is too long");
+        if (hrp.length() < 1)
+            throw new IllegalArgumentException("human-readable part is too short: " + hrp.length());
+        if (hrp.length() > 83)
+            throw new IllegalArgumentException("human-readable part is too long: " + hrp.length());
```

Comment:
I think we could make an exception (or new rule) to allow `if (cons) throw exception` on 1-line for argument checking.

Reply:
In the context of this PR, this is no longer necessary.